### PR TITLE
Updating new rbac

### PR DESCRIPTION
**Pull request by amit9192**

### CODE REVIEW

File: docs/Learn/RBAC.md [hunk 1]
```diff
@@ -1,21 +1,129 @@
...
 
-Roles categorize users and define what permissions those users have, such as what data they can read or what they can modify.
+## Komodor Roles
+Roles are built from one or more policies.
 
-### Komodor Roles:
+### Built-in Roles
+- account-admin - has full permissions on the account
+- viewer - has access to view all resources on the account
+- developer - has access to view all resources and perform basic actions (TODO: link to actions page)
```
Comment (michaeldeift) [inline docs/Learn/RBAC.md:12]: fix the TODO

File: docs/Learn/RBAC.md [hunk 2]
```diff
@@ -1,21 +1,129 @@
...
+    - Deletion of Pods  
+    - Scaling all supported resource types   
+    - Restarting all supported resource types   
+```
+[
+    {
+        "actions": [
+            "view:all"
+            "delete:pod",
+            "scale:*",
```
Comment (michaeldeift) [inline docs/Learn/RBAC.md:80]: action:* is not supported yet

File: docs/Learn/RBAC.md [hunk 3]
```diff
@@ -1,21 +1,129 @@
...
+]
+```
+
+2 - The following policy allows viewing all resources and managing all Komodor configurations for all clusters  
+```
+[
+    {
+        "actions": [
+            "view:all"
+            "manage:*"
```
Comment (michaeldeift) [inline docs/Learn/RBAC.md:108]: same here for *

File: docs/Learn/Actions.md [hunk 4]
```diff
@@ -84,6 +84,12 @@ Using Komodor you can run multiple actions against your resource. You'll be able
...
     verbs:
     - patch
+  - apiGroups:
+    - ""
+    resources:
+    - pods/eviction
+    verbs:
+    - create
 ```
 - **Please note:** At this stage, only account **Admins** can see and perform actions against their resources, in the near future we will add the ability to configure granular permissions
```
Comment (michaeldeift) [inline docs/Learn/Actions.md:94]: Maybe change this note to support also new rbac

File: docs/Learn/RBAC.md [hunk 5]
```diff
@@ -1,21 +1,129 @@
-## Role-Based Access Control
+# Role-Based Access Control
 
-### Intro
+## Intro
+Komodor allows assigning users with Roles to control their access and permissions (such as what data they can read or what they can modify).
```
Comment (michaeldeift) [inline docs/Learn/RBAC.md:4]: Maybe better:
"or which resource they can modify"

File: docs/Learn/RBAC.md [hunk 6]
```diff
@@ -1,21 +1,129 @@
...
 
-Roles categorize users and define what permissions those users have, such as what data they can read or what they can modify.
+## Komodor Roles
+Roles are built from one or more policies.
 
-### Komodor Roles:
+### Built-in Roles
+- account-admin - has full permissions on the account
```
Comment (michaeldeift) [inline docs/Learn/RBAC.md:10]: Add some note about account-admin. Must be at least one in org, and the last one can't be removed

File: docs/Learn/RBAC.md [hunk 7]
```diff
@@ -1,21 +1,129 @@
...
+### Policy Creation
+You can easily create new policies using the Komodor platform  
+
+1. Access the settings page  
+<img src="./img/settings.png" width="250">  
+2. Go to the Policies page  
+<img src="./img/policies.png" width="250">    
+3. Add policy   
+You will now enter a policy creation wizard, you can easily manage your policies with it
+<img src="./img/policy-wizard.png" width="650">
```
Comment (michaeldeift) [inline docs/Learn/RBAC.md:128]: Maybe add the full flow of:
1. Create policy
2. Create role
3. Assign to user
4. Add bulk users with certain role

File: docs/Learn/Actions.md (hunk 4 above)
Comment (guymenahem) [reply docs/Learn/Actions.md:94]: +1

Comment (amit9192) [reply docs/Learn/Actions.md:94]: good idea

**Comment by amit9192** [comment]
@michaeldeift added the fixes

File: docs/Learn/RBAC.md [hunk 8]
```diff
@@ -1,21 +1,131 @@
...
+You can easily create new policies using the Komodor platform  
+
+1. Access the settings page  
+<img src="./img/settings.png" width="250">  
+2. Go to the Policies page  
+<img src="./img/policies.png" width="250">    
+3. Add policy   
+You will now enter a policy creation wizard, you can easily manage your policies with it
+<img src="./img/policy-wizard.png" width="650"> 
+4. Create a role associated with this policy
```
Comment (michaeldeift) [inline docs/Learn/RBAC.md:129]: add images

**Comment by amit9192** [comment]
@michaeldeift added images

**Review by michaeldeift — APPROVED** [review]

**Pull request merged**